frontend/src/metabase-types/api/settings.ts

```diff
@@ -1,8 +1,32 @@
+export interface FormattingSettings {
+  "type/Temporal"?: DateFormattingSettings;
```

Comment:
This is just our existing API contract for the settings, it has nothing to do with semantic types.